test/apicoverage-webhook/ignoredfields.yaml

```diff
@@ -0,0 +1,23 @@
+- package: serving.v1alpha1
```

Comment:
For `ko` to load this, you'll want to move it into a `kodata/` subdirectory next to your `package main`.